ferveo-python/test/test_serialization.py

```diff
@@ -38,6 +39,10 @@ def make_shared_secret():
     pass
 
 
+def make_pk():
+    return Keypair.random().public_key()
+
+
```

Comment:
Can the commented-out TODOs in this test file now be uncommented due to the updated bindings?

Reply:
I've tried to update some of these tests but I didn't implement `__richcmp__` for all types involved, so I'm going to cut some corners here. Ideally, we would have all classes methods implemented, but that's a bit of work and not really worth the effort at this point IMHO.